fxgl-entity/src/test/kotlin/com/almasb/fxgl/entity/EntityTest.kt

```diff
@@ -170,14 +170,15 @@ class EntityTest {
         assertThat(entity.components, hasItem(control))
     }
 
+    // Test seems duplicate, see l.51 `Add component fails if same type already exists`
```

Comment:
Same here. Maybe change this to create two `TestControl` objects and check if the first one is the added component like:
```
control1 = TestControl()
control2 = TestControl()
entity.addComponent(control1)
entity.addComponent(control2)
assertThat(entity.getComponent(TestControl::class.java),  `is`(control1))
```